nix/devshell.nix

```
@@ -1,39 +1,23 @@
-{ pkgs }:
+{ pkgs ? import <nixpkgs> { } }:
```

The uses a Nix channel, which is not pinned and hopefully this feature will soon be deprecated. For reproducibility, we should continue using the Flake as source: 

```suggestion
{ pkgs }:
```

Flakes are the future of Nix, if the feature is not enabled on your Nix installation yet, here is how to: https://nixos.wiki/wiki/Flakes

Also if Nix Flakes are not enabled, the old `nix shell` command should still work as [a compatibility layer](https://github.com/edolstra/flake-compat) is configured